src/components/code-editor/examples/code-editor-composite.tsx

```
@@ -0,0 +1,98 @@
+                    this.invalid
+                        ? 'This code is invalid'
+                        : 'The code should be in JSON format'
+                }
+                value={this.value}
+                required={this.required}
+                invalid={this.invalid}
+                disabled={this.disabled}
+                readonly={this.readonly}
+                onChange={this.handleChange}
+                language="json"
+            />,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)